templates/repo/issue/filters.tmpl

```diff
@@ -38,7 +38,9 @@
 					{{range .Labels}}
 						{{$exclusiveScope := .ExclusiveScope}}
 						{{if and (ne $previousExclusiveScope $exclusiveScope)}}
-							<div class="divider"></div>
+							{{if or (not .IsArchived) (and .IsArchived $.ShowArchivedLabels)}}
```

Comment:
This is only a partial fix as it will be buggy when you have a scope with an archived label and an active label, and the checkbox is not selected.
In that case, the divider will be missing even though it shouldn't.